JetLagged/app/build.gradle.kts

```diff
@@ -34,9 +34,9 @@ android {
     }
 
     signingConfigs {
```

Comment:
Maybe we just remove this block altogether? I think Android Studio already defaults to the local key with these parameters for debug builds.

Reply:
Question is if the `benchmark` buildType allows it, but I assume there's default `signingConfig` even if it's not defined.

Reply:
Benchmark requires release build, and release requires a certificate :-/ Not sure how else to work around this